components/camel-pulsar/src/main/java/org/apache/camel/component/pulsar/configuration/PulsarConfiguration.java

```diff
@@ -55,6 +55,10 @@
     private long ackGroupTimeMillis = 100;
     @UriParam(label = "consumer", defaultValue = "LATEST")
     private SubscriptionInitialPosition subscriptionInitialPosition = LATEST;
+    @UriParam(label = "consumer", description = "Maximum number of times that a message will be redelivered before being sent to the dead letter queue. If this value is not set, no Dead Letter Policy will be created")
+    private Integer maxRedeliverCount;
+    @UriParam(label = "consumer", description = "Name of the topic where the messages which fail {maxRedeliverCount} times will be sent. Note: if not set, default topic name will be <topicname>-<subscriptionname>-DLQ")
```

Comment:
Lets avoid using {xxx} in the description but just the name of the option is fine

Reply:
Also avoid the < > in the DLQ name as the generated docs in the json will strip out invalid characters.